SU-HSDO/suhumsci
# shs-108: override inherited styles to fix college name menu overlap

## Summary
This introduces the actual branch I intended to merge for SHS-108 and also reverts the previous work that was merged in. 

## Steps to Test
1. Go to the homepage from the link below and go down to mobile size in browser and make sure the text doesn't overflow onto the 'Menu' button.
2. Do this for both colorful/traditional
3. Open the menu at mobile and ensure that it isn't all messed up (it would be obvious - this is what the prior in progress work was doing)

## Notes
* Feel free to check the previous PR files to ensure reversion went correctly: https://github.com/SU-HSDO/suhumsci/pull/1188/files

## Screenshots
![Screen Shot 2022-10-04 at 1 53 29 PM](https://user-images.githubusercontent.com/8405274/193891080-7403c3c4-4c6a-475c-9b2d-45dc3428604b.png)
